packages/devtools_app/lib/src/screens/inspector/inspector_service.dart

```diff
@@ -951,46 +954,46 @@ abstract class ObjectGroupBase implements Disposable {
         // different and filter that way.
         continue;
       }
-      if (field.isConst && field.isStatic) {
+      if (field.isConst! && field.isStatic!) {
         properties[field.name] = field.declaredType;
       }
     }
     return properties;
   }
 
-  Future<SourcePosition> getPropertyLocation(
+  Future<SourcePosition?>? getPropertyLocation(
```

Comment:
I don't think the future needs to be nullable

Reply:
done